cluster/definition.go

```diff
@@ -305,28 +305,33 @@ func (d *Definition) UnmarshalJSON(data []byte) error {
 		return errors.New("unsupported version")
 	}
 
+	*d = def
+
+	return nil
+}
+
+// VerifyHashes returns an error if hashes populated from json object doesn't matches actual hashes.
+func (d Definition) VerifyHashes() error {
```

Comment:
`VerifyDefinitionHashes` since we will add `VerifyLockHashes` and both method appear on the Lock struct